core/templates/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -113,20 +111,15 @@ angular.module('oppia').directive('questionsList', [
           ctrl.directiveSubscriptions = new Subscription();
           var _reInitializeSelectedSkillIds = function() {
             ctrl.selectedSkillId = ctrl.getSelectedSkillId();
-            if (ctrl.selectedSkillId !== null) {
```

Comment:
Is this an expected change for this PR? Why remove it?

Reply:
This code was not needed anymore because now `selectedSkillId` is used for fetching skills.

Reply:
Oh interesting.. I think it was a list because questions could have multiple skills linked to it. Can you please check that questions with multiple skills get listed as expected in this directive when used in the topic editor? Thanks!